README.md

```diff
@@ -194,11 +194,25 @@ lgtm query \
 	-q=/path/to/query.ql
 ```
 
+### Get results from a query ID
+
+```bash
+lgtm x-list-query-results XXXXXXXXXXXXXXXXXXX
+```
+
+#### Examples
+
+##### Get projects name
+```bash
+lgtm x-list-query-results XXXXXXXXXXXXXXXXXXX | 
+jq '.[].Project.displayName' | tr -d '"'
+```
+
```

Comment:
I'd like to put experimental commands' documentation in a separate 2 heading after the `## LGTM-CLI usage`.

Reply:
No problem, do you want me to open a separate PR for that?

Reply:
You could put it together with the other README update (the legal section thing).